apps/indexer/src/api/services/proposals/proposals.ts

```diff
@@ -23,24 +24,32 @@ export class ProposalsService {
     limit = 10,
     orderDirection = "desc",
     status,
+    status_in,
     fromDate,
   }: ProposalsRequest): Promise<DBProposal[]> {
     const proposals = await this.proposalsRepo.getProposals(
       skip,
       limit,
       orderDirection,
       status,
+      status_in,
       fromDate,
```

Comment:
Logic flaw: When both status and status_in are provided, status takes precedence and status_in is ignored. This creates inconsistent API behavior. Either make these parameters mutually exclusive with validation, or define clear precedence rules in the API documentation.